src/super_gradients/common/sg_loggers/base_sg_logger.py

```diff
@@ -348,3 +351,29 @@ def _save_code(self):
             self.add_file(name)
             code = "\t" + code
             self.add_text(name, code.replace("\n", "  \n  \t"))  # this replacement makes tb format the code as code
+
+    def _sanitize_checkpoint(self, state_dict: dict) -> dict:
```

Comment:
Can we use a softer term, please... 
sanitize sounds like DictConfigs are bacteria.
Maybe,`_convert_to_native`?

Reply:
To some extent it is. Naming was intentional to indicate that we do a state dict cleanup to contain primitive types only. Maybe in future we will add even more strict checking here.

Reply:
@ofrimasad I mean if it's a show-stopper then I can rename it.
But the word is pretty common in SD https://github.com/search?q=sanitize&type=repositories